templates/featured/includes/featured_message.part.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+
+<p class="featured-message">
+    {% if featured_message.hook != None and featured_message.hook != '' %}
+        <span class="important home-hook">{{ featured_message.hook }}</span>
+    {% endif %}
+    {% if featured_message.hook != None and featured_message.message != '' %}
```

Comment:
Ce n'est pas plutôt `{% if featured_message.message != None and featured_message.message != '' %}` ? C'est pas bien de copier/coller... :P